src/io.rs

```diff
@@ -84,14 +209,41 @@ mod tests {
                 .write_record("The Quick Brown Fox".as_bytes())
                 .unwrap();
         }
+        {
+            let mut af = File::open(actual_filename).unwrap();
+            let mut ef = File::open(expected_filename).unwrap();
+            let mut actual = vec![0; 0];
+            let mut expected = vec![0; 0];
+            af.read_to_end(&mut actual).unwrap();
+            ef.read_to_end(&mut expected).unwrap();
+
+            assert_eq!(actual, expected);
+        }
 
-        let mut af = File::open(actual_filename).unwrap();
-        let mut ef = File::open(expected_filename).unwrap();
-        let mut actual = vec![0; 0];
-        let mut expected = vec![0; 0];
-        af.read_to_end(&mut actual).unwrap();
-        ef.read_to_end(&mut expected).unwrap();
+        let _ = std::fs::remove_file(actual_filename);
```

Comment:
sneak in a line to clean up test files

Reply:
I am not sure but perhaps we can also create some file in some temp folder? But this seems working as well.